fedimint-server/src/net/api.rs

```diff
@@ -55,6 +56,8 @@ pub async fn run_server(
 
     debug!(addr = cfg.api_bind_addr, "Starting WSServer");
     let server = ServerBuilder::new()
+        .max_connections(cfg.max_connections)
+        .ping_interval(Duration::from_secs(10))
```

Comment:
From the docs:

> This option is used to keep the connection alive, and is just submitting Ping frames, without making any assumptions about when a Pong frame should be received.

So does this actually help with closing inactive connections?

Reply:
Yes, if the ping fails then the connection will be closed as far as I can tell:

https://github.com/paritytech/jsonrpsee/blob/master/server/src/transport/ws.rs#L475-L496

Reply:
I would expect it does help in a lot of __normal__ situations, as it will trigger the destination or NAT in front of it to respond with some ICMP and that will close a lingering connection (e.g. due to NAT evicting it, or the host closing it without sending `FIN` packet).

However if everything is going to go through Tor, I'm not sure how this behaves. If we expect all the connections to be transient for privacy reasons, maybe we should do something smarter on the higher level and e.g. disconnect everyone after fixed time no matter what.